pynicotine/nowplaying.py

```diff
@@ -300,7 +300,7 @@ def other(self, command):
             return None
 
         try:
-            output = execute_command(command, returnoutput=True, hidden=True)
+            output = execute_command(command, returnoutput=True, hidden=True).decode()
             self.title["nowplaying"] = output
```

Comment:
Can we have

`            self.title["nowplaying"] = output.decode()`

That is the same effect but easier to read.

Reply:
Done.